gateway-modules/apim-ballerina-configs/configs/org/wso2/carbon/apimgt/gateway/microgateway/microUtil.bal

```diff
@@ -0,0 +1,213 @@
+package org.wso2.carbon.apimgt.gateway.microgateway;
+import ballerina.lang.errors;
+import ballerina.lang.system;
+import ballerina.lang.jsons;
+import org.wso2.carbon.apimgt.gateway.dto as dto;
+import org.wso2.carbon.apimgt.gateway.holders as holders;
+import ballerina.lang.files;
+import ballerina.lang.blobs;
+import org.wso2.carbon.apimgt.ballerina.util;
+import org.wso2.carbon.apimgt.gateway.utils as gatewayUtil;
+import org.wso2.carbon.apimgt.gateway.constants as Constants;
+errors:TypeCastError err;
+
+function loadConfigs () {
+
+    string name = system:getEnv(Constants:GW_HOME);
+    files:File t = {path:name + "/microgateway/microConf.json"};
+
+    if (files:exists(t)) {
+        try {
+            //files:File t = {path:name + "/microgateway/microConf.json"};
+            //not working in windows                                     ????????????????
```

Comment:
Let's add these logs in a proper way. Maybe we can use todo and say that we need to fix the following.